thundra/constants.py

```diff
@@ -56,6 +56,8 @@
 THUNDRA_DISABLE_MONGO_INTEGRATION = 'thundra_agent_lambda_trace_integrations_mongodb_disable'
 THUNDRA_DISABLE_SQLALCHEMY_INTEGRATION = 'thundra_agent_lambda_trace_integrations_sqlalchemy_disable'
 THUNDRA_DISABLE_CHALICE_INTEGRATION = 'thundra_agent_lambda_trace_integrations_chalice_disable'
+THUNDRA_DISABLE_HTTP_4XX_ERROR = 'thundra_agent_trace_integrations_http_set_error_on_4xx_response_code_disable'
```

Comment:
Instead of two different configuration, I think we can use one as minimum status code to consider a status code as error and its name can be `thundra_agent_trace_integrations_http_error_status_code_min`